api/conf/settings/base.py

```diff
@@ -60,6 +61,11 @@
     },
 ]
 
+# Allow both during transition. We need to deploy this with zero-downtime, so we can
+# deploy the WSGI app at first and then switch the configuration over to use uvicorn
+# and the ASGI app after it is deployed. In the mean time, async routes will still be
+# "async" just inside their own event loop per-request.
+ASGI_APPLICATION = "conf.asgi.application"
```

Comment:
After we deploy the API using gunicorn + uvicorn to use the ASGI application, we'll be able to delete the WSGI_APPLICATION setting and the wsgi.py file.